prototype/p2p/network_lifecycle.go

```diff
@@ -0,0 +1,61 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"strings"
+)
+
+func (g *P2PModule) configure(protocol, address, external string, peers []string) {
+	g.protocol = protocol
+	g.address = address
+	g.externaladdr = external
+	g.peerlist = types.NewPeerlist()
+
+	// this is a hack to get going no more no less
+	for i, p := range peers {
+		peer := types.NewPeer(uint64(i+1), p)
+		port := strings.Split(peer.Addr(), ":")
+		myport := strings.Split(g.address, ":")
+		if port[1] == myport[1] {
+			g.id = peer.Id()
+		}
+		g.peerlist.Add(*peer)
+	}
+}
+
+func (g *P2PModule) init() error {
+	msg := Message(int32(0), 1, types.PocketTopic_P2P_PING, "", "")
```

Comment:
Why do we only need to register one type of message? Maybe add a comment what this is doing.

Reply:
It's only type of message we got atm